src/content/1.7/modules/core_updates/1.7.7.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Changes in PrestaShop 1.7.7
+menuTitle: Changes in 1.7.7
+---
+
+# Notable changes in PrestaShop 1.7.7
+
```

Comment:
Can you please add a mention about jQuery?

Reply:
The page is not finished of course, it's focues on hooks but jquery info will be added in another PR